api/api_full.go

```diff
@@ -862,7 +862,7 @@ type FullNode interface {
 	RaftState(ctx context.Context) (*RaftStateData, error) //perm:read
 	RaftLeader(ctx context.Context) (peer.ID, error)       //perm:read
 
-	// IPC-specific methods //
+	// IPCAgent-specific methods //
```

Comment:
This is not correct. This are IPC-specific methods to call in Lotus, nothing to do with IPC agent.
```suggestion
	// IPC-specific methods //
```

Reply:
Fixed